### ISSUE TITLE
Analyses of defenses

### ISSUE THREAD

**Issue by anishathalye** [yenchenlin/awesome-adversarial-machine-learning#6]
ML security, or any security field in general, is going to have cases where papers make a certain claim, and later, that claim ends up being invalidated. For example, we once thought [MD4](https://en.wikipedia.org/wiki/MD4) was a secure hash function, but now it's known to be [broken](https://en.wikipedia.org/wiki/MD4#Security). This is a natural thing to happen in a security field: future analysis showing that a certain scheme is not secure.

For this reason, it might be a good idea for a resource that lists defenses against adversarial examples to also keep up-to-date with future analyses of those defenses, so that readers looking at this list do not mistakenly believe that a broken scheme is secure.

The list currently has a number of defenses that are known to have true robustness significantly below what is claimed in the paper (in most cases, the true accuracy is 0%). This includes:

* [Defensive Distillation](https://arxiv.org/abs/1511.04508) (broken by [this paper](https://arxiv.org/abs/1608.04644))
* [PixelDefend](https://arxiv.org/abs/1710.10766) (broken by [this paper](https://arxiv.org/abs/1802.00420))

Do you think the list could be organized so that it's clear to readers that while defense papers are interesting, for certain papers, later analyses have shown that the schemes can be circumvented? ([robust-ml.org](https://www.robust-ml.org/defenses/) is one example of such an organization)

**Comment by yenchenlin** [comment]
Good idea! I will start to add up-to-date future analyses to the list two weeks later (after I finish my mandatory military service).

Also, as the number of papers related to adversarial examples has exploded over the past two years, I am thinking about writing an annotated bibliographies (also served as a suggested reading list) [here](https://github.com/yenchenlin/awesome-adversarial-machine-learning/blob/master/bibliographies.md) for people new to the field to track the status.

Feel free to share your thoughts here!

**Comment by yenchenlin** [comment]
Will keep this issue open until the up-to-date analyses are added.
